library/src/signer.ts

```diff
@@ -0,0 +1,10 @@
+import { ApiActions } from './constants/api-actions.enum'
+import { BlossomMessages } from './messages/blossom-messages'
+
+export class Signer {
+  constructor(private messages: BlossomMessages) {}
+
+  public signMessage(message: string): Promise<string> {
+    return this.messages.sendMessage(`${ApiActions.SIGNER}.signMessage`, message)
```

Comment:
I would keep all actions in the enum as so far, in this case with `signer.signMessage`